cloud-sql/mysql/client-side-encryption/tests/test_encrypt_and_insert_data.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+import pytest
+
+from snippets.cloud_kms_env_aead import init_tink_env_aead
+from snippets.cloud_sql_connection_pool import init_db
+from snippets.encrypt_and_insert_data import encrypt_and_insert_data
+
+
+REQUIRED_ENV_VARS = [
+    "MYSQL_USER",
+    "MYSQL_PASSWORD",
+    "MYSQL_DATABASE",
+    "MYSQL_HOST",
+    "CLOUD_KMS_KEY"]
+
+table_name = f"votes_{uuid.uuid4().hex}"
+
+
+@pytest.fixture
+def check_env_vars():
+    for var in REQUIRED_ENV_VARS:
```

Comment:
Instead of checking for these, is there a reason why they can't be set as part of the test and then torn down?

Reply:
These are values being used to configure how the test are run. The test can't set them because it doesn't know what the values should be. 

This is just provides a more descriptive error message when running the tests than using `varname = os.environ[var]`